tests/IsoCodes/Tests/SsnTest.php

```diff
@@ -77,15 +136,35 @@ public function testValidSsn($ssn)
      */
     public function testInvalidSsn($ssn)
     {
-        $this->assertFalse($this->ssn->validate($ssn));
```

Comment:
> you cannot force them to change their code just because you want to impose a new implementation to serve your projects.

I would say "force" by prevent that this method will not be available. To avoid surprise on major release.

BTW I insist, according to [Semantic Versioning Specification](http://semver.org/) about minor version: 

> Minor version Y (x.Y.z | x > 0) MUST be incremented if **new, backwards compatible functionality is introduced** to the public API. It MUST be incremented if any public API functionality **is marked as deprecated**. It MAY be incremented if substantial new functionality or improvements are introduced within the private code. It MAY include patch level changes. Patch version MUST be reset to 0 when minor version is incremented.

It's exactly what is done on this PR. Backward compatible features and deprecation marks.

So I don't understand what is the problem about put it on new minor version.

Even so, if you still don't want it as is, I propose two other "solutions":
 
1. Remove `trigger_error` calls but I don't recommend at all. BTW this was accepted on #38.
2. Remove BC safe and wait for 2.0. Same thing, user won't be noticed of breaking change before upgrading.